modules/ROOT/pages/writing.adoc

```diff
@@ -646,12 +710,16 @@ Before the import starts, the following schema query is being created:
 CREATE INDEX ON :Person(surname)
 ----
 
-*Take into consideration that the first label is used for the index creation.*
+The name of the created index is `spark_INDEX_<LABEL>_<NODE_KEYS>`, where `<LABEL>` is the first label from the `labels` option and `<NODE_KEYS>` is a dash-separated sequence of one or more properties as specified in the `node.keys` options.
+In this example, the name of the created index is `spark_INDEX_Person_surname`.
+If the `node.keys` option were set to `"name,surname"` instead, the index name would become `spark_INDEX_Person_name-surname`.
+
+The index is recreated if it is already present.
```

Comment:
I haven't checked what happens in this case, does it not use the `IF NOT EXISTS` clause in this case?

Reply:
Checked this, there is a logic to check if an equivalent index exists before creating the index, so shouldn't recreate one.

Reply:
When I tried it, there was no error and the creation time of the index was updated. Have you tried as well?